package.json

```diff
@@ -19,8 +19,9 @@
   "dependencies": {
     "ethjs-util": "^0.1.3",
     "jsontokens": "^0.6.5",
+    "mnid": "^0.1.1",
     "nets": "^3.2.0",
-    "uport-lite": "^0.2.4"
+    "uport-lite": "^1.0.0-alpha-2"
```

Comment:
@zachferland Try changing this to `next` instead of the version number. I think that should automatically get the latest ones.